contributing/documentation/building_the_manual.rst

```diff
@@ -120,7 +120,7 @@ If you run into errors, you may try the following command:
 
 If you get a ``MemoryError`` or ``EOFError``, you can remove the ``classes/`` folder and
 run ``make`` again.
-This will drop the class references from the final HTML documentation but will keep the
+This will drop the class references from the final HTML documentation, but will keep the
```

Comment:
Not the best at grammar so checking if this one feels, while short, like two independent clauses.

1. This will drop the class references from the final HTML documentation
2. This will keep the rest intact

Reply:
I think the comma is fine here.